scripts/deploy/nlp/deploy_ray_hf.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+import signal
+import sys
+from typing import Optional
+
+from nemo.deploy.deploy_ray import DeployRay
+from nemo.deploy.nlp.hf_deployable_ray import HFRayDeployable
+
+LOGGER = logging.getLogger("NeMo")
+
```

Comment:
Do you think it makes sense to merge the ray and pytriton based deployment in one script and make the deploy option an argument?

Same thing can be done for the query as well.